tests/integrationtests/tst_perfparser.cpp

```diff
@@ -223,17 +223,21 @@ private slots:
         QTest::addColumn<QString>("errorMessagePart");
         QTest::addColumn<int>("waitTime");
 
-        QTest::addRow("pre-exported perfparser") << QFINDTESTDATA("file_content/true.perfparser") << QString() << 2000;
-        QTest::addRow("invalid data") << QFINDTESTDATA("tst_perfparser.cpp") << QStringLiteral("invalid perf data file")
-                                      << 1000;
-        QTest::addRow("PERF v1") << QFINDTESTDATA("file_content/perf.data.true.v1")
-                                 << QStringLiteral("invalid perf data file") << 1000;
+        const auto perfData = QFINDTESTDATA("file_content/true.perfparser");
```

Comment:
these hunks should be in the parent patch, or?

Reply:
No because:
* only the new checks "the real patch here" do output the file name in the error message
* the new patch adjusts the error messages as per your requests (most were not checked at all), so we can't check them before the patch

If you want to then I can add to the "pre"-testing an 'expected' "can't parse the file if it doesn't match the expected filename", but I don't think that's anything useful to test.
Please note if there are other things you want to be included in the previous commit.

Reply:
what I don't understand: the first patch introduces

```
       QTest::addRow("pre-exported perfparser") << QFINDTESTDATA("file_content/true.perfparser") << QString() << 2000;
```

then this patch here changes it to:

```
        const auto perfData = QFINDTESTDATA("file_content/true.perfparser");
        QTest::addRow("pre-exported perfparser") << perfData << QString() << 2000;
```

this part should be cleaned up and implemented directly in this way in the initial patch such that there is no change hunk at all here.

dito for the "PERF v1" case below

Reply:
alright, let's leave it as is then